### PR TITLE
Next

### PR DESCRIPTION
**Pull request by nsenave**
- update dependencies

### CODE REVIEW

File: src/main/java/fr/insee/pogues/configuration/StaticResourcesForFOPConfig.java [hunk 1]
```diff
@@ -20,10 +20,16 @@
 @Component
 public class StaticResourcesForFOPConfig {
 
-    final static Logger logger = LogManager.getLogger(StaticResourcesForFOPConfig.class);
+    final static Logger LOG = LogManager.getLogger(StaticResourcesForFOPConfig.class);
```
Comment (nsenave) [inline src/main/java/fr/insee/pogues/configuration/StaticResourcesForFOPConfig.java:23]: this boilerplate could be replaced with a lombok annotation at class level like `@Slf4j`

File: src/main/java/fr/insee/pogues/configuration/properties/ApplicationProperties.java
Comment (nsenave) [inline src/main/java/fr/insee/pogues/configuration/properties/ApplicationProperties.java:1]: The breakdown of properties in several objects is great!

**Review by nsenave — COMMENTED** [review]
some comments

File: src/main/java/fr/insee/pogues/configuration/StaticResourcesForFOPConfig.java (hunk 1 above)
Comment (laurentC35) [reply src/main/java/fr/insee/pogues/configuration/StaticResourcesForFOPConfig.java:23]: You have an example somewhere of usage `@Slf4j` annotation ?
Thanks

Comment (laurentC35) [reply src/main/java/fr/insee/pogues/configuration/StaticResourcesForFOPConfig.java:23]: I found this : https://medium.com/@AlexanderObregon/enhancing-logging-with-log-and-slf4j-in-spring-boot-applications-f7e70c6e4cc7

Perfect

**Comment by nsenave** [comment]
still work in progress, moving it to another branch

**Pull request closed without merging**